CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [@mattrglobal/pairing-crypto-rn 0.5.0](https://github.com/mattrglobal/pairing_crypto/compare/0.4.2...0.5.0)
+
+Updates `"react-native"` from version 0.68.0 to 0.72.8
```

Comment:
Maybe also mention `Dependency upgrades and vulnerability fixes`, I see there are a few dependency upgrades

Reply:
Fixed